CHANGELOG.md

```diff
@@ -0,0 +1,35 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [0.22.0] - 2024-01-11
+
+### Added
+- `PromptImageGenerationEngine` for generating images from text prompts. 
+- `VariationImageGenerationEngine` for generating variations of an input image according to a text prompt.
+- `InpaintingImageGenerationEngine` for modifying an input image according to a text prompt within the bounds of a mask defined by a mask image. 
+- `OutpaintingImageGenerationEngine` for modifying an input image according to a text prompt outside the bounds of a mask defined by a mask image.
+- `ImageArtifact` for storing image data; used heavily by the image Engines, Tasks, and Drivers.
+- `ImageLoader` for loading images files into `ImageArtifact`s.
+- Support for all Tokenizers in `OpenAiChatPromptDriver`, enabling OpenAi drop-in clients such as TogetherAi.
```

Comment:
"enabling OpenAi drop-in" -> "enabling OpenAI drop-in"
"TogetherAI" -> "Together AI"